### DISCUSSION TITLE
Holding a torch or spider eye to enable light mod, and other requests.

### DISCUSSION THREAD

**Discussion by saikoyama** [SchmarrnDevs/Lighty#50] [Ideas and Suggestions]
It would be nice if this mod worked like it does with an option in the shader pack "Complimentary Reimagined". Holding a torch or a spider eye in either hand would enable the light level display.

I like this mod because the block light display is subtle and not distracting. It would also be nice if there was an option to change the light display from carpets to a flat color on the blocks.

Using a mod like this helps me make sure that monsters do not spawn without me needing to spam torches or change my lighting settings from smooth to blocky.

**Comment by andi-makes** [comment]
Thank you for your suggestions!

The torches one sounds like a great thing to have and the flat mode shouldn't be too difficult to implement.

I'll look into implementing both, and keep you updated in this thread. Again, thanks for the suggestions!

**Comment by andi-makes** [comment]
This has been implemented for a while now (I cannot exactly remember when I added the feature), so I'll close this discussion.

You can enable the behavior in the Lighty Settings by enabling "Auto on".

Thanks again for your suggestion, it is a really convenient feature :)
